projects/plugins/debug-helper/modules/class-protect-helper.php

```diff
@@ -0,0 +1,354 @@
+
+		if ( isset( $_GET['show_notice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			add_action( 'admin_notices', array( $this, 'display_notice' ) );
+		}
+
+		$settings = self::get_stored_settings();
+		if ( $settings['overwrite_status'] ) {
+			// Display a notice when this module overwrites Protect Status.
+			add_action( 'admin_notices', array( $this, 'display_protect_overwritten_notice' ) );
+
+			$option_name      = Automattic\Jetpack\Protect\Status::OPTION_NAME;
+			$option_time_name = Automattic\Jetpack\Protect\Status::OPTION_TIMESTAMP_NAME;
+			add_filter( "option_$option_name", array( $this, 'get_mock_response' ) );
+			add_filter( "option_$option_time_name", array( $this, 'filter_option_time' ) );
+
+			if ( 'error' === $settings['status'] ) {
+				add_filter( 'pre_http_request', array( $this, 'filter_status_fetch' ), 10, 3 );
+			}
+		}
+
+	}
+
+	/**
+	 * Add submenu item.
+	 */
+	public function register_submenu_page() {
+		add_submenu_page(
+			'jetpack-debug-tools',
+			'Protect Helper',
+			'Protect Helper',
+			'manage_options',
+			'protect-helper',
+			array( $this, 'render_ui' ),
+			99
+		);
+	}
+
+	/**
+	 * Renders the UI.
+	 */
+	public function render_ui() {
+		$settings = $this->get_stored_settings();
+		?>
+		<h1>Protect Helper</h1>
+
+		<form action="<?php echo esc_url( admin_url( 'admin-post.php' ) ); ?>" method="post">
+
+		<table class="form-table" role="presentation">
+		<tbody>
+		<tr>
+			<th scope="row">Overwrite Protect Status</th>
+			<td>
+				<fieldset>
+					<p>If enable, Protet status fetched from the server will be ignored and the rules below will define the current status.</p>
+					<label><input type="radio" name="overwrite_status" value="1" <?php echo ( $settings['overwrite_status'] ? 'checked="checked"' : '' ); ?>> enabled</label><br>
+					<label><input type="radio" name="overwrite_status" value="0" <?php echo ( ! $settings['overwrite_status'] ? 'checked="checked"' : '' ); ?>> disabled</label><br>
+				</fieldset>
+			</td>
+		</tr>
+		<tr>
+			<th scope="row"><label for="idc_siteurl">Status</label></th>
+			<td>
+				<select name="status">
+					<option value="empty" <?php selected( 'empty', $settings['status'] ); ?>>Empty - When the initial scan did not run yet</option>
+					<option value="error" <?php selected( 'error', $settings['status'] ); ?>>Error - When there's an error fetching the status</option>
+					<option value="incomplete" <?php selected( 'incomplete', $settings['status'] ); ?>>Incomplete - When we have results, but some extensions were not scanned</option>
+					<option value="complete" <?php selected( 'complete', $settings['status'] ); ?>>Complete - When we have results for all extensions</option>
+				</select>
+			</td>
+		</tr>
+		<tr>
+			<th scope="row">Vulnerabilities</th>
+			<td>
+				<label><input type="checkbox" name="vuls_for_core" <?php echo ( $settings['vuls_for_core'] ? 'checked="checked"' : '' ); ?>> Add vulnerabilities to core</label><br>
```

Comment:
```suggestion
				<label><input type="checkbox" name="vuls_for_core" <?php echo ( $settings['vuls_for_core'] ? 'checked="checked"' : '' ); ?>> Add vulnerabilities to Core</label><br>
```
Very minor suggestion - capitalize "Core" to match "Themes" and "Plugins" in the same list.